qa-tests-backend/src/main/groovy/util/Helpers.groovy

```diff
@@ -137,10 +136,11 @@ class Helpers {
         log.debug "Will scan ${image} to ${saveName}"
 
         try {
-            Path imageScans = Paths.get(Env.QA_TEST_DEBUG_LOGS).resolve("image-scans")
+            Path p = Paths.get(Env.QA_TEST_DEBUG_LOGS)
+            Path imageScans = p.resolve('image-scans')
             new File(imageScans.toAbsolutePath().toString()).mkdirs()
 
-            Process proc = "./scripts/ci/roxctl.sh image scan -i ${image} -a".execute(null, new File(".."))
+            Process proc = "./scripts/ci/roxctl.sh image scan -i ${image} -a".execute([], new File(".."))
```

Comment:
with `null` compiler could not what method call. Passing a list solves the problem.